force-app/main/default/lwc/propertyTile/__tests__/propertyTile.test.js

```diff
@@ -0,0 +1,73 @@
+import { createElement } from 'lwc';
+import PropertyTile from 'c/propertyTile';
+
+const PROPERTY_DETAILS = {
+    City__c: 'Some City',
+    Beds__c: '3',
+    Baths__c: '1',
+    Price__c: '450000',
+    Thumbnail__c: 'some-property.jpg'
+};
+
+describe('c-property-tile', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    it('displays a property in the tile', () => {
+        const element = createElement('c-property-tile', {
+            is: PropertyTile
+        });
+        element.property = PROPERTY_DETAILS;
+        document.body.appendChild(element);
+
+        const headerEl = element.shadowRoot.querySelector('.truncate');
+        expect(headerEl.textContent).toBe(PROPERTY_DETAILS.City__c);
+
+        const pElement = element.shadowRoot.querySelector('p');
```

Comment:
We're calling element vars whateverEl in all the repos - would you mind to follow that rule for consistency?

Reply:
Changes are done to maintain naming consistency